src/main/java/org/opentripplanner/routing/services/FareService.java

```diff
@@ -1,13 +1,19 @@
 package org.opentripplanner.routing.services;
 
+import org.opentripplanner.api.model.Leg;
 import org.opentripplanner.routing.core.Fare;
 import org.opentripplanner.routing.spt.GraphPath;
 
+import java.util.List;
+
 /**
  * Computes a fare for a given GraphPath.
  * @author novalis
  *
  */
 public interface FareService {
-	public Fare getCost(GraphPath path);
+	Fare getCost(GraphPath path);
+    default Fare getCost(GraphPath path, List<Leg> legs) {
+        return getCost(path);
+    }
```

Comment:
Fix indent?

Reply:
Weird... this looks fine in my IDE.

Reply:
Fixed by reintending it. Must have been a tab issue.